Profiler/Trace-Script.ps1

```diff
@@ -210,21 +210,21 @@ function Trace-Script {
 
         $lineNumber = $hit.Line
         if (-not $file.Lines.ContainsKey($lineNumber)) {
-            $lineProfile = [PSCustomObject] @{
-                Percent      = 0
-                HitCount     = 0 
-                Duration     = [TimeSpan]::Zero
-                Average      = [TimeSpan]::Zero
-                SelfDuration = [TimeSpan]::Zero
-                SelfAverage  = [TimeSpan]::Zero
+            $lineProfile = [Profiler.LineProfile] @{
+                # Percent      = 0
```

Comment:
Not sure what's up with that, seen it in your issue tasks, but I don't know what is your question. :) For me just delete the commented out code.

Reply:
Was waiting to see if you'd comment that it should be a constructor. Ironically I haven't profiled it yet to measure how much of a perf benefit.

Reply:
I don't think there will be huge difference between object initializer and creating an object and populating it's properties. I would avoid constructor params, but since this will probably move to c# and it was fast enough before in pure powershell let's just remove the comments and merge it.

Reply:
👍 You're welcome to do it now if you want to merge ASAP. If not I'll do it tonight.